compiler/rustc_codegen_llvm/src/coverageinfo/mod.rs

```diff
@@ -17,14 +17,16 @@ use rustc_middle::mir::coverage::CoverageKind;
 use rustc_middle::mir::Coverage;
 use rustc_middle::ty::layout::HasTyCtxt;
 use rustc_middle::ty::Instance;
+use rustc_target::abi::Align;
 
 use std::cell::RefCell;
 
 pub(crate) mod ffi;
 pub(crate) mod map_data;
 pub mod mapgen;
 
-const VAR_ALIGN_BYTES: usize = 8;
+// Can be replaced with `Option::unwrap()` once that is a const fn
```

Comment:
```suggestion
// FIXME: Replace with `Option::unwrap()` once that is a const fn
```

Reply:
Alternatively, I would be fine with replacing this constant with a helper function (named something like `align_to_8_bytes`) that just uses `.unwrap()` internally.

In this context it doesn't specifically *need* to be `const`.

Reply:
In fact, you can probably just put `Align::from_bytes(8).unwrap()` at the two use sites. The constant isn't really achieving much here; we're just copying a hardcoded `llvm::Align(8)` from clang anyway.